src/main/java/bio/terra/service/filedata/google/firestore/FireStoreFileDao.java

```diff
@@ -91,6 +124,7 @@ private DocumentSnapshot lookupByFileId(Firestore firestore,
 
     // See comment in FireStoreUtils.java for an explanation of the batch size setting.
     private static final int DELETE_BATCH_SIZE = 500;
+
     void deleteFilesFromDataset(Firestore firestore, String datasetId, InterruptibleConsumer<FireStoreFile> func)
         throws InterruptedException {
 
```

Comment:
Would it be possible to log which batch we are on during delete, since we are logging the increment in 500 already?

Reply:
Alas, no. We are streaming from firestore and dont know the total size